test/lender/Lender.sol

```diff
@@ -176,7 +176,7 @@ contract Lender {
     return IElement(e).swap(swap, fund, r, d);
   }
 
-  /// @dev lend method signature for pendle
+  /// @dev lend method signature for pendleAddr
```

Comment:
nah, thats being weird. this is for `Pendle` on a high level. Pendle as a product - as a protocol. focusing on the *Addr implementation detail is being over-specific